docs/topics/multiplatform-mobile/multiplatform-mobile-setup.md

```diff
@@ -37,12 +37,14 @@ We recommend that you install the latest stable versions for compatibility and b
    </tr>
    <tr>
         <td><a href="multiplatform-mobile-plugin-releases.md">Kotlin Multiplatform Mobile plugin</a></td>
-        <td>In Android Studio, select <strong>Settings/Preferences | Plugins</strong>, search <strong>Marketplace</strong> for <i>Kotlin Multiplatform Mobile</i>, and then install it.</td>
+        <td>In Android Studio, select <menupath>Settings/Preferences | Plugins</menupath>, search <control>Marketplace</control> for <i>Kotlin Multiplatform Mobile</i>, and then install it.</td>
    </tr>
    <tr>
         <td><a href="plugin-releases.md#update-to-a-new-release">Kotlin plugin</a></td>
-        <td><p>The Kotlin plugin should be compatible with the Kotlin Multiplatform Mobile plugin. Refer to the <a href="multiplatform-mobile-plugin-releases.md#release-details">compatibility table</a>.</p>
-            <p>To update the plugin, in Android Studio navigate to <strong>Tools | Kotlin | Configure Kotlin Plugin Updates</strong> and then select the latest version in the <strong>Stable</strong> channel.</p></td>
+        <td>
+            <p>The Kotlin plugin is bundled with each Android Studio release. However, it's still needs to be updated to the latest version to avoid compatibility issues.</p> 
```

Comment:
```suggestion
            <p>The Kotlin plugin is bundled with each Android Studio release. However, it still needs to be updated to the latest version to avoid compatibility issues.</p> 
```